specification/nginx/resource-manager/NGINX.NGINXPLUS/stable/2023-04-01/examples/Configurations_CreateOrUpdate.json

```diff
@@ -1,11 +1,15 @@
~{
~  "parameters": {
~    "api-version": "2023-04-01",
~    "subscriptionId": "00000000-0000-0000-0000-000000000000",
     "resourceGroupName": "myResourceGroup",
     "deploymentName": "myDeployment",
     "configurationName": "default",
+<<<<<<< HEAD
+    "requestBody": {
+=======
```

Comment:
could you please fix the merge issues in this folder

Reply:
Thanks for capturing, fixed.